fullstory-segment-middleware/build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
     }
 
     compileOnly 'com.fullstory:instrumentation-full:1.23.0@aar'
-    compileOnly 'com.segment.analytics.android:analytics:4.7.1'
+    compileOnly 'com.segment.analytics.android:analytics:4.10.4'
```

Comment:
would it make sense to just use something like `4+`

Reply:
updated it to use the prefix range, thank you for the guidance.

reference: https://docs.gradle.org/current/userguide/single_versions.html